.github/workflows/build.yaml

```diff
@@ -0,0 +1,89 @@
+name: Build
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: ["contribute"]
```

Comment:
Action trigger criteria is very intentional here: builds are not triggered for every push unless the change is made to an important shared branch such as `contribute` (the default development branch.) It is possible to specify branch patterns here. For example, if we wanted to trigger a build for every DRTSIM commit: `branches: ["contribute", "DRTSIM-*"]`.

There is potential here to aid in a movement towards a git ops workflow, where project viewers, releases, release notes are all driven through git operations.